src/containers/User/UserArea.tsx

```diff
@@ -305,11 +327,47 @@ const MainMenuBar: React.FC<MainMenuBarProps> = ({
             </Dropdown>
           </List.Item>
         )}
+        {isMobile && (
+          <List.Item onClick={() => logout()}>
+            <div id="menu-divider" className="ui divider"></div>
+            <Dropdown text={t('MENU_USER_OPTIONS')}>
+              <Dropdown.Menu>
+                {templates
+                  .filter(({ templateCategory: { uiLocation } }) => {
+                    return uiLocation.includes(UiLocation.User)
+                  })
+                  .map(({ code, name, icon, templateCategory: { icon: catIcon } }) => (
+                    <Dropdown.Item
+                      key={code}
+                      icon={icon || catIcon}
+                      text={name}
+                      onClick={() => push(`/application/new?type=${code}`)}
+                    />
+                  ))}
+                {/* TODO: Add language selector for mobile... */}
+                <Dropdown.Item icon="log out" text={t('MENU_LOGOUT')} onClick={() => logout()} />
+              </Dropdown.Menu>
```

Comment:
I couldn't see how the modal worked for the language selector.  Maybe because I only had one language to choose from?

Reply:
You can enable more languages from "Configuration -> Localisations"

I've fixed this in https://github.com/openmsupply/conforma-web-app/pull/1578 anyway :)